package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "_s",
+  "version": "1.0.0",
+  "scripts": {
+    "css-build": "npm run css-lint && npm run css-compile && npm run css-prefix && npm run css-format && npm run css-fix",
+    "css-lint": "stylelint style.css",
+    "css-compile": "node-sass sass/style.scss style.css --output-style expanded --indent-type tab --indent-width 1",
+    "css-prefix": "postcss style.css -u autoprefixer -o style.css --no-map",
+    "css-format": "stylefmt style.css",
+    "css-fix": "stylelint style.css --fix",
+    "scss-lint": "stylelint sass/**/*.scss",
```

Comment:
You need to specify that you are using the SCSS syntax here by adding `--syntax scss` to this command

It should also use the `stylelint-config-wordpress/scss` shared config for SCSS syntax per WPCS for SCSS